src/app/Components/Form/Form.module.css

```diff
@@ -0,0 +1,36 @@
+.form {
+  display: grid;
+}
+
+.input__text {
+  display: grid;
+  margin: 0.2em 0 0.4em;
+  font-size: 0.6em;
+  text-transform: uppercase;
+}
+
+.button {
+  border-radius: 1em;
+  border: 0;
+  padding: 0.6em;
+  background-color: var(--text-color);
+  color: var(--text-color-contrast);
+  font-family: var(--font-family-global);
+  transition: background 0.4s;
+}
+
+.button:hover {
+  background: var(--text-color-contrast);
+  color: var(--text-color);
+}
+
+.input__field {
+  opacity: 70%;
```

Comment:
I would check if this is required, because the background already has an opacity.